script.js

```diff
@@ -220,7 +219,7 @@ function addSegmentBtnListener() {
     };
 
     // Track the event using Segment with the computed property names
-    analytics.track(titleInput.textContent, trackEvent);
+    analytics.track(actionInfo.eventName, trackEvent);
```

Comment:
replaced the tracking event name with whatever is in the constant for it, instead of the title input